spec/lib/miq_automation_engine/miq_ae_service_spec.rb

```diff
@@ -116,6 +116,8 @@ module MiqAeServiceSpec
           Datacenter
           ManageIQ::Providers::BaseManager
           ManageIQ::Providers::PhysicalInfraManager
+          ManageIQ::Providers::Lenovo::PhysicalInfraManager
+          ManageIQ::Providers::Lenovo::Provider
```

Comment:
I agree, the point of this test was to ensure the automate service models get created when new rails models are introduced.

@juliancheal You should be able to follow the existing pattern to create these service models.  Let me know if you need any help here.